testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/theme/CustomAccountResourceProviderTest.java

```diff
@@ -0,0 +1,27 @@
+package org.keycloak.testsuite.theme;
+
+import org.keycloak.testsuite.AbstractTestRealmKeycloakTest;
+import org.keycloak.theme.Theme;
+import org.keycloak.representations.idm.RealmRepresentation;
+import org.junit.Test;
+
+public class CustomAccountResourceProviderTest extends AbstractTestRealmKeycloakTest {
+
+    @Override
+    public void configureTestRealm(RealmRepresentation testRealm) {
+
+    }
+
+    @Test
+    public void testProviderOverride() {
```

Comment:
It should be better to make sure we can reach out to the `/account` endpoint and look at the response from the custom provider.

We could add tests at `AccountRestServiceTest` that configured the custom theme to a realm and then query the `/account` endpoint. Wdyt?

Reply:
Added a test in `AccountRestServiceTest` that does this.